pennylane_lightning/lightning_qubit.py

```diff
@@ -197,13 +197,13 @@ def stopping_condition(self):
         and observable) and returns ``True`` if supported by the device."""
 
         def accepts_obj(obj):
-            if obj.name == "QFT" and len(obj.wires) >= 10:
-                return False
-            if obj.name == "GroverOperator" and len(obj.wires) >= 13:
-                return False
-            if getattr(obj, "has_matrix", False):
-                return not (qml.operation.is_trainable(obj))
-            return obj.name in self.observables.union(self.operations)
+            if obj.name == "QFT" and len(obj.wires) < 10:
+                return True
+            if obj.name == "GroverOperator" and len(obj.wires) < 13:
+                return True
+            return not isinstance(obj, qml.tape.QuantumTape) and getattr(
+                self, "supports_operation", lambda name: False
+            )(obj.name)
```

Comment:
I assume this is `((not tape) and supported)` and not  `(not (tape and supported))`. If 1, I'd like a parenthesis around the statement to ease-the-eyes if possible.

Reply:
You are entirely correct. I just did it!